lib/expansion_rules.rb

```diff
@@ -63,6 +64,7 @@ def details_fields(*fields)
   STEP_BY_STEP_FIELDS = (DEFAULT_FIELDS + [%i(details step_by_step_nav title), %i(details step_by_step_nav steps)]).freeze
   TRAVEL_ADVICE_FIELDS = (DEFAULT_FIELDS + details_fields(:country, :change_description)).freeze
   WORLD_LOCATION_FIELDS = [:content_id, :title, :schema_name, :locale, :analytics_identifier].freeze
+  FACET_FIELDS = %i[content_id title schema_name details].freeze
```

Comment:
Is there any way you can be specific with which `details` you need? using the `details_fields(...)` example like above? Just because it avoids unnecessary dependency resolution.

Reply:
Yes good point, though we currently want to expose everything in the details hash it could well expand and nothing here would explicitly restrict it.

Reply:
@thomasleese https://github.com/alphagov/publishing-api/pull/1463/commits/0531adcd4be7c90803f0849a3419b97993cb678d I've been explicit about what to expand.